src/ctap/test_helpers/env.rs

```diff
@@ -0,0 +1,49 @@
+use crate::clock::CtapInstant;
+use crate::ctap::command::{
+    AuthenticatorAttestationMaterial, AuthenticatorConfigParameters,
+    AuthenticatorVendorConfigureParameters, Command,
+};
+use crate::ctap::data_formats::ConfigSubCommand;
+use crate::ctap::status_code::Ctap2StatusCode;
+use crate::ctap::{key_material, Channel, CtapState};
+use crate::env::Env;
+
+// In tests where we define a dummy user-presence check that immediately returns, the channel
+// ID is irrelevant, so we pass this (dummy but valid) value.
+const DUMMY_CHANNEL: Channel = Channel::MainHid([0x12, 0x34, 0x56, 0x78]);
+
+pub struct SetupEnterpriseAttestationResponse {
```

Comment:
Why do we want a single use struct here?

Reply:
I'm not sure I understand this question. This struct is mainly for grouping all outputs of the setup so that unittests/fuzzers could access?

Reply:
Oh I think I get your question. It only has one field now, why don't just return AttestationMaterial? I think we can do that too, but we might want return type naming to be consistent when we add more setup methods in the future?

Reply:
Why not directly return `AuthenticatorAttestationMaterial`? Do you intend to extend `setup_enterprise_attestation` and this struct helps with that? Are there other functions that will reuse `SetupEnterpriseAttestationResponse`? Or is this data type going to implement any helpful functions for fuzzing?

Reply:
Ah we responded at the same time. :) I'd say if consistent naming is the only reason, remove it.

Reply:
done